packages/run-protocol/src/vpool-xyk-amm/stableSwapCurve.js

```diff
@@ -0,0 +1,388 @@
+// @ts-check
+import { assert, details as X } from '@agoric/assert';
+import { Nat } from '@agoric/nat';
+import { AmountMath } from '@agoric/ertp';
+import {
+  makeRatio,
+  floorMultiplyBy,
+  makeRatioFromAmounts,
+  floorDivideBy,
+} from '@agoric/zoe/src/contractSupport/index.js';
+
+const A = 85;
+const BASIS_POINTS = 10000n;
+const maxLoopLimit = 1000;
```

Comment:
we like constants to be all caps.
```suggestion
const MAX_LOOP_LIMIT = 1000;
```